cpp/cmake_modules/BuildUtils.cmake

```diff
@@ -240,7 +240,9 @@ function(ADD_ARROW_LIB LIB_NAME)
       DEPENDENCIES
       SHARED_INSTALL_INTERFACE_LIBS
       STATIC_INSTALL_INTERFACE_LIBS
-      OUTPUT_PATH)
+      OUTPUT_PATH
+      SHARED_INSTALL_EXTRA_TARGETS
+      STATIC_INSTALL_EXTRA_TARGETS)
```

Comment:
Do we actually use any of the changes here anymore?

Reply:
Removed.

Reply:
I'm still confused here, now the arguments were added back but we don't use them?